api/src/main/java/com/messagebird/RequestValidator.java

```diff
@@ -0,0 +1,113 @@
+package com.messagebird;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier.BaseVerification;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.auth0.jwt.exceptions.SignatureVerificationException;
+import com.auth0.jwt.interfaces.Clock;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.auth0.jwt.interfaces.JWTVerifier;
+import com.messagebird.exceptions.RequestValidationException;
+
+/**
+ * RequestValidator validates webhook signature signed by MessageBird services.
+ */
+public class RequestValidator {
```

Comment:
There is an "ExampleRequestSignatureValidation" class in the examples folder to show customers how to implement it, you should also delete the class and write an example usage class for RequestValidator.

Reply:
I have updated that class